app/controllers/certificates_controller.rb

```diff
@@ -1,4 +1,6 @@
 class CertificatesController < ApplicationController
+  before_action :authenticate_user!
```

Comment:
we no longer need this since you added the `before_action` on the `ApplicationController` which this controller inherits from